mods/ts/sequences/voxels.yaml

```diff
@@ -120,57 +120,118 @@ sonic:
 	idle:
 	turret: sonictur
 
-#truk: # TODO: unused
-#	idle:
+truk: # unused
+	idle:
 
-#truck2: # TODO: unused
-#	idle:
+truck2: # unused
+	idle:
 
-#utnk: # TODO: unused alternative version of Devil's Tongue
+#utnk: # unused alternative version of Devil's Tongue
 #	idle:
 
-#mrj: # TODO: unused
-#	idle:
-#	turret: mrjtur
+mrj: # unused
+	idle:
+	turret: mrjtur
 
-#m113: # TODO: unused
-#	idle:
-#	turret: m113tur
+m113: # unused
+	idle:
+	turret: m113tur
 
-#mlrs: # TODO: unused
-#	idle:
-#	turret: mlrstur
+mlrs: # unused
+	idle:
+	turret: mlrstur
 
-#arty: # TODO: unused
-#	idle:
-#	barrel: artybarl
+arty: # unused
+	idle:
+	barrel: artybarl
 
-#1tnk: # TODO: unused
+#jeep: # unused and .vxl and .hva don't match up
 #	idle:
-#	turret: 1tnktur
-#	barrel: 1tnkbarl
 
-#2tnk: # TODO: unused
-#	idle:
-#	turret: 2tnktur
-#	barrel: 2tnkbarl
+1tnk: # unused
+	idle:
+	turret: 1tnktur
+	barrel: 1tnkbarl
 
-#3tnk: # TODO: unused
-#	idle:
-#	turret: 3tnktur
-#	barrel: 3tnkbarl
+2tnk: # unused
+	idle:
+	turret: 2tnktur
+	barrel: 2tnkbarl
 
-#ftnk: # TODO: unused
-#	idle:
+3tnk: # unused
+	idle:
+	turret: 3tnktur
+	barrel: 3tnkbarl
 
-#tire: # TODO: unused debris
-#	idle:
+ftnk: # unused
+	idle:
 
-#probe: # TODO: unused
-#	idle:
+tire: # unused debris
+	idle:
 
-#piece: # TODO: unused debris
+#probe: # unused and .vxl and .hva don't match up
 #	idle:
 
+pulscan: # unused
```

Comment:
Defining all these unused voxels, adding extra VRAM and loading time overheads, for the sake of an asset browser feature that most players will never see, is really not a good idea!